xla/debug_options_flags.cc

```diff
@@ -287,6 +287,8 @@ DebugOptions DefaultDebugOptionsIgnoringFlags() {
   opts.set_xla_gpu_cudnn_gemm_max_plans(5);
 
   opts.set_xla_gpu_enable_triton_gemm_int4(false);
+
+  opts.set_xla_experimental_ignore_channel_id(true);
```

Comment:
If it's experimental, the default should be false, no?

Reply:
I thought we'd want to ignore channel ids by default. If that's the case, defaulting it to true would make more sense. Or we just dont call it experimental at all.

Reply:
ok i changed the default to false for now.